src/RcppExports.cpp

```diff
@@ -24,3 +24,13 @@ BEGIN_RCPP
     return rcpp_result_gen;
 END_RCPP
 }
+
```

Comment:
Why all these changes to `src` files?

Reply:
It seemed an older way to do the Rcpp integration as Rcpp no longer generates the files this way. The file was added over 4 years ago https://github.com/Displayr/flipRegression/commit/896bf576874d6aefcc2992ef4e5b71c903ca399f   and was using Kevin Ushey's package `Kmisc` to generate the plumbing for the Rcpp integration. The `Kmisc` package has been removed from CRAN since 2018, https://cran.r-project.org/web/packages/Kmisc/index.html. The code is added to RcppExports via Rcpp in the modern version. It seemed like a good way to make sure the code doesn't get out of sync with any Rcpp improvements.